lbry/wallet/wallet.py

```diff
@@ -269,11 +269,7 @@ def read(self):
 
     def upgrade(self, json_dict):
         json_dict = json_dict.copy()
-        version = json_dict.pop('version', -1)
-        if version == -1:
-            pass
-        upgraded = self._default.copy()
-        upgraded.update(json_dict)
```

Comment:
This changes `upgraded` and `upgraded` is discarded. Perhaps you meant to return it instead of `json_dict`.